lib/ChangeTypes.js

```diff
@@ -21,9 +21,9 @@ module.exports = {
   // e.g. x * 2 -> 2x
   REARRANGE_COEFF: 'REARRANGE_COEFF',
   // e.g. x ^ 0 -> 1
-  REDUCE_EXPONENT_BY_ZERO: 'REDUCE_EXPONENT_BY_ZERO',
+  REMOVE_EXPONENT_BY_ZERO: 'REDUCE_EXPONENT_BY_ZERO',
```

Comment:
Oh woops, I meant to rename both to make it uniform. I don't want to switch between `REDUCE` and `REMOVE`.